acceptance/tests/options/config_file/command_line_override.rb

```diff
@@ -14,28 +17,30 @@
 EOM
 
   agents.each do |agent|
-    step "Agent #{agent}: create config file" do
-      config_dir = agent.tmpdir("config_dir")
-      config_file = File.join(config_dir, "facter.conf")
+    config_dir = get_default_fact_dir(agent['platform'], on(agent, facter('kernelmajversion')).stdout.chomp.to_f)
+    config_file = File.join(config_dir, "facter.conf")
+
+    step "Agent #{agent}: create config file in default location" do
+      on(agent, "mkdir -p '#{config_dir}'")
       create_remote_file(agent, config_file, config)
```

Comment:
This file should be reset to it original state (deleted in this case) as part of a teardown step to ensure that a modified file does not leak into other tests.